.tekton/main-build.yaml

```
@@ -58,6 +58,8 @@ spec:
     secret:
       secretName: '{{ git_auth_secret }}'
 
+  # See comment in operator-bundle-build.yaml for spec.timeouts; adjustments of this timeout
+  # here might might require tweaking the timeout for the operator bundle as well.
```

While this looks okay, I suggest being more instructive/prescriptive so that engineers don't take the action as optional because, well, it isn't.

How about this?
```suggestion
  # IMPORTANT: when changing timeouts here, read and follow timeout instructions in operator-bundle-build.yaml.
```

Also, I suggest copying this comment to timeouts in all other `.tekton/*-build.yaml`-s because it's inevitable we'll tweak things as time passes.

---

Done in commit ff9e444027.